docsite/docs/trouble_shooting_guides/long_path_issue.md

```diff
@@ -0,0 +1,28 @@
+---
+sidebar_position: 1
+---
+
+# Long Path Issue On Windows 10
+
+If you are encountering an error with Ninja on Windows due to long paths not being enabled typically means that the path to your project or files is too long for Ninja to handle. Windows has a limitation on the maximum length of file paths, and if your project's path exceeds this limit, Ninja may fail to execute properly. 
```

Comment:
it might be nice to display a bit of the error that they get:
```
error Failed to install the app. Command failed with exit code 1: gradlew.bat app:installDebug -PreactNativeDevServerPort=8081 FAILURE: Build failed with an exception. * What went wrong: Execution failed for task ':react-native-vision-camera:buildCMakeDebug[arm64-v8a]'. > com.android.ide.common.process.ProcessException: ninja: Entering directory
```